File: algorithm/src/main/java/baekjoon/p04999/Main.java
package baekjoon.p04999;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.next(), sc.next()));
    }

    private static String solve(String a, String b) {
        if (a.length() >= b.length()) {
            return "go";
        } else {
            return "no";
        }
    }

}

File: algorithm/src/main/java/baekjoon/p04999/Main.kt
package baekjoon.p04999

fun main(args: Array<String>) {
    println(solve(readLine()!!, readLine()!!))
}

private fun solve(a: String, b: String): String {
    return when (a.length >= b.length) {
        true -> "go"
        false -> "no"
    }
}